test/fixtures/transaction-response.json

```diff
@@ -0,0 +1,89 @@
+{
+  "type": "payment",
+  "address": "r9cZA1mLK5R5Am25ArfXFmqgNwjZgnfk59",
+  "specification": {
+    "source": {
+      "address": "r9cZA1mLK5R5Am25ArfXFmqgNwjZgnfk59",
+      "amount": {
+        "currency": "XRP",
+        "value": "1.112209"
```

Comment:
Why is this missing counterparty ? Elsewhere I see an empty string used ? I'm not all that familiar with ripple-rest, but glancing over this.